CMakeLists.txt

```diff
@@ -4,6 +4,13 @@ project( googletest-distribution )
 
 enable_testing()
 
+include(GNUInstallDirs)
+
+option(SKIP_GOOGLETEST_INSTALL "Skip installation" ${SKIP_GOOGLETEST_INSTALL} )
+if( NOT SKIP_GOOGLETEST_INSTALL )
+   set( ENABLE_GOOGLETEST_INSTALL ON )
```

Comment:
This seems excessive; why not just use `if(NOT SKIP_GOOGLETEST_INSTALL)`? (Or make the option `ENABLE_GOOGLETEST_INSTALL` and default it to `ON`.)